recipes-wpe/wpeframework/include/wpeframework.inc

```diff
@@ -1,25 +1,14 @@
-# WPE Framework common
-HOMEPAGE = "https://github.com/rdkcentral"
+# Common for thunder and tools
 SECTION = "wpe"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=f1dffbfd5c2eb52e0302eb6296cc3711"
+HOMEPAGE = "https://github.com/rdkcentral/Thunder"
 
-PV = "3.0+gitr${SRCPV}"
-
-PACKAGECONFIG      ??= ""
-PACKAGECONFIG[debug] = "-DCMAKE_BUILD_TYPE=Debug,-DCMAKE_BUILD_TYPE=Release,"
+SRC_URI[md5sum] = "42b518b9ccd6852d1d709749bc96cb70"
+SRC_URI[sha256sum] = "f3c45b121cf6257eafabdc3a8008763aed1cd7da06dbabc59a9e4d2a5e4e6697"
+SRC_URI = "git://github.com/rdkcentral/Thunder.git;protocol=git;branch=master"
+SRCREV = "3e7c6f09af0a495ca51caa564370fb2c4f23c39e"
 
+PR = "r0"
+PV = "3.0+git${SRCPV}"
 S = "${WORKDIR}/git"
-
```

Comment:
The WPEFRAMEWORK_PERSISTENT_PATH and WPEFRAMEWORK_EXCEPTIONS_ENABLE should be moved to the wpeframework.bb file
The rest I belive should end in wpeframework-common.inc? I believe we should leave those special flags at least in the backward compatibility mode

Reply:
WPEFRAMEWORK_EXCEPTIONS_ENABLE is moved to wpeframework.bb file. WPEFRAMEWORK_PERSISTENT_PATH still need to kept in the common.inc since it is used by plugins (currently ocdm)